substrate/primitives/consensus/slots/src/lib.rs

```diff
@@ -135,6 +136,7 @@ impl From<Slot> for u64 {
 	Eq,
 	TypeInfo,
 )]
+#[repr(transparent)]
 pub struct SlotDuration(u64);
```

Comment:
Does this change the `PassBy` that is required for the RuntimeAPI?  
I assume not. But maybe it would have made more sense to use `PassByInner` from the beginning 🙈

Reply:
`PassBy` is required by the runtime interface, aka when calling into the host from the runtime api. I don't know any host function that uses `Slot` nor `SlotDuration` :P